### PR TITLE
[docs] Fix formatting and indentation

### PR DESCRIPTION
**Pull request by andrejtokarcik**

### CODE REVIEW

File: docs/5.0/config-reference.md [hunk 1]
```diff
@@ -199,13 +199,13 @@ auth_service:
...
-#    nodes to be upgraded to 4.4
-#
-#    "node-sync" : sessions recording will be streamed from node -> auth -> storage service
-#    "proxy-sync : sessions recording will be streamed from proxy -> auth -> storage service
-#
+    # Proxy and node send logs directly to S3 or other storage without
+    # storing the records on disk at all. *-sync requires all nodes to be
+    # upgraded to 4.4.
+    #
+    #    "node-sync" : sessions recording will be streamed from node -> auth -> storage service
```
Comment (webvictim) [inline docs/5.0/config-reference.md:206]:
```suggestion
    #    "node-sync" : session recordings will be streamed from node -> auth -> storage service
```

File: docs/5.0/config-reference.md [hunk 2]
```diff
@@ -199,13 +199,13 @@ auth_service:
...
+    #    "proxy-sync : sessions recording will be streamed from proxy -> auth -> storage service
```
Comment (webvictim) [inline docs/5.0/config-reference.md:207]:
```suggestion
    #    "proxy-sync : session recordings will be streamed from proxy -> auth -> storage service
```

File: docs/5.0/config-reference.md [hunk 3]
```diff
@@ -351,26 +351,26 @@ proxy_service:
...
-     # to limit access to applications
-     labels:
-        env: "prod"
-     # Optional Dynamic Labels
-     commands:
-     - name: "os"
-       command: ["/usr/bin/uname"]
-       period: "5s"
+    enabled: yes
+    # We've a small debug app that can be used to make sure application
```
Comment (webvictim) [inline docs/5.0/config-reference.md:355]:
```suggestion
    # Teleport contains a small debug app that can be used to make sure Application
```

File: docs/5.0/config-reference.md [hunk 4]
```diff
@@ -351,26 +351,26 @@ proxy_service:
...
+    # Access is working correctly. It'll output JWTs so it can be useful
```
Comment (webvictim) [inline docs/5.0/config-reference.md:356]:
```suggestion
    # Access is working correctly. The app outputs JWTs so it can be useful
```

**Review by webvictim — APPROVED** [review]
LGTM; also added a few other changes

**Review by klizhentas — APPROVED** [review]

**Pull request merged**